.github/workflows/ci.yaml

```diff
@@ -340,11 +365,41 @@ jobs:
       # https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/
       id-token: write
     steps:
+<<<<<<<
```

Comment:
Your conflict resolution failed I think

Reply:
Removed the update to repoconfig 0.10 for now.  So this PR takes it only upto 0.9.2

Reply:
Oh, I guess you were already bitten by the flaky migration script.